R/properties.R

```diff
@@ -138,22 +138,47 @@ addQuotesOrComma <- function(v, term)
 }
 #' Copy attributes from one object to another
 #'
-#' Copies the "label", "name", "question" and "questiontype" attributes
-#' for each for variable in a \code{\link{data.frame}}.
+#' Copies attributes such as "label", "name", "question" and "questiontype"
+#' from one object to another. If both objects are data frames,
+#' variables in the recipient data frame will also receive attributes from
+#' variables with the same name in the donor data frame.
 #' @param data.without.attributes an object to receive attributes from, such as
 #' a data.frame, list, or matrix
 #' @param data.with.attributes an object to copy attributes from
 #' @param attr.to.not.copy character vector of attribute names appearing in
 #' \code{data.with.attributes} that should not be copied
 #' @return A copy of \code{data.without.attributes} with all the attributes
 #' of \code{data.with.attributes}.
-#' @details In the case when both arguments are \code{data.frame}s, any attributes
+#' @details In the case when both arguments are data frames, any attributes
```

Comment:
Can you please also add a note that the function is recursive and will copy attributes when the inputs are nested lists?

Reply:
OK done to both your comments